config.js

```diff
@@ -2,12 +2,13 @@ var web3 = require('web3');
 var net = require('net');
 
 var config = function () {
-  
+
   this.logFormat = "combined";
   this.provider = new web3.providers.HttpProvider("http://localhost:8545");

```

Comment:
Instead of this mess of having a baked in `config.js` here, can somebody consider: https://github.com/ewasm/etherchain-light/pull/18#issuecomment-414149508